global.json

```diff
@@ -9,6 +9,6 @@
     "rollForward": "latestMinor"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.23122.4"
```

Comment:
we should not move to this version, @brettfo we need to stick to 7 until ready for .NET 8 release, right?

Reply:
Correct, keep this at 7.